### PR TITLE
Toolkit Items for Scheduling Interviews

### PR DESCRIPTION
**Pull request by bernars-usa**
Closes https://github.com/usds/OPM_pilot/issues/536. This PR adds four email templates (two for applicants, two for SMEs, along with a couple of scheduling workbooks for schedulers.

@humancompanion-usds: When you get a chance, can you add a SVG for email templates to include in the toolkit? Probably a little envelope. I'll leave a line comment where I'm referencing it.

@stephaniefaith as FYI

### CODE REVIEW

File: _includes/toolkit/phone-assessment-interviews.html [hunk 1]
```diff
@@ -7,6 +7,30 @@
...
+  <li class="chp-toolkit__item">
+    <a href="{{site.baseurl}}/toolkit/phone-assessment-interviews/email-template-sme-availability/" class="chp-toolkit__link">{% include svg/file-email.svg %}Email to SMEs for Interview Availability
+    </a>
+  </li>
+  <li class="chp-toolkit__item">
+    <a href="{{site.baseurl}}/toolkit/phone-assessment-interviews/example-sme-availability-worksheet/" class="chp-toolkit__link">{% include svg/file-excel.svg %}SME Interview Availability Worksheet
+    </a>
+  </li>
+  <li class="chp-toolkit__item">
+    <a href="{{site.baseurl}}/toolkit/phone-assessment-interviews/sme-recusal-email-template/" class="chp-toolkit__link">{% include svg/file-email.svg %}SME Recusal Email Template
```
Comment (bernars-usa) [inline _includes/toolkit/phone-assessment-interviews.html]: > `{% include svg/file-email.svg %}`

@humancompanion-usds: Here's one of the four spots where I'm referencing a new email template SVG that doesn't exist yet. I'm not sure where you got the other ones.

Comment (humancompanion-usds) [reply _includes/toolkit/phone-assessment-interviews.html]: I have a local copy of font awesome. I will pull this down and update.

Comment (bernars-usa) [reply _includes/toolkit/phone-assessment-interviews.html]: Thanks! ✨ ✉️

File: pages/phases/phone-assessment-interviews/scheduling.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ intro: To schedule applicants, work with a team of schedulers or admins who can
   {{ page.intro }}
 </p>
 
-Before you [schedule training](training.md) for the first round of interviews, divide the SMEs into two groups based on which round of interviews they will conduct. Consider how many applicants passed resume review, as well as any scheduling conflicts you may be aware of at this time. Send SMEs calendar invites for the days or weeks they'll be conducting interviews as soon as possible.
+Before you [schedule training](training.md) for the first round of interviews, divide the SMEs into two groups based on which round of interviews they will conduct. Consider how many applicants passed resume review, as well as any scheduling conflicts you may be aware of at this time. Send SMEs calendar invites for the days or weeks they'll be conducting interviews as soon as possible. These invites should just be a placeholder notification of when interviews will generally take place. To solicit the specific times that SMEs will make available for interviews during this period, send them the <a href="{{site.baseurl}}/toolkit/phone-assessment-interviews/email-template-sme-avialability/">SME Availability Email</a>.
```
Comment (humancompanion-usds) [inline pages/phases/phone-assessment-interviews/scheduling.md]: email-template-sme-whatnow? Check spelling of availability in the link itself.

Comment (bernars-usa) [reply pages/phases/phone-assessment-interviews/scheduling.md]: Haha! Avialiablibity, it's a thing!

**Comment by humancompanion-usds** [comment]
This looks good. Just a clarifying question: Will there be applicant templates for those not moving forward and those moving onto the 2nd interview?

**Comment by bernars-usa** [comment]
> Will there be applicant templates for those not moving forward and those moving onto the 2nd interview?

Ah, good point. I was thinking about the NOR codes for those not moving forward, but let me look at clarifying these email templates so they can be used for first and second interviews.

**Pull request merged**